src/lib/operators/sort.hpp

```diff
@@ -38,17 +47,19 @@ class Sort : public AbstractReadOnlyOperator {
       const std::shared_ptr<AbstractOperator>& copied_input_right) const override;
   void _on_set_parameters(const std::unordered_map<ParameterID, AllTypeVariant>& parameters) override;
 
-  // The operator is separated in three different classes. SortImpl is the common templated implementation of the
-  // operator. SortImpl* und SortImplMaterializeOutput are extra classes for the visitor pattern. They fulfill a certain
-  // task during the Sort process, as described later on.
   template <typename SortColumnType>
   class SortImpl;
+
   template <typename SortColumnType>
   class SortImplMaterializeOutput;
 
-  std::unique_ptr<AbstractReadOnlyOperatorImpl> _impl;
-  const ColumnID _column_id;
-  const OrderByMode _order_by_mode;
+  std::shared_ptr<const Table> _get_materialized_output(const std::shared_ptr<PosList>& pos_list);
```

Comment:
"get" sounds like a cheap step. Maybe "materialize_output"?

Reply:
Fixed in 2e4a1e739bcc9286a783cba00336ca8e61325125